tools/export_onnx.py

```diff
@@ -19,8 +21,8 @@ def make_parser():
     parser.add_argument(
         "--output-name", type=str, default="yolox.onnx", help="output name of models"
     )
-    parser.add_argument("--input", default="images", type=str, help="input name of onnx model")
-    parser.add_argument("--output", default="output", type=str, help="output name of onnx model")
+    parser.add_argument("--input", default="images", type=str, help="input node name of onnx model")
+    parser.add_argument("--output", default="output", type=str, help="output node name of onnx model")
```

Comment:
This line is too line(more than 100 col), please lint as PEP8 suggested.

Reply:
Nice catch, Updated.